tools/pipelines/templates/include-test-real-service.yml

```
@@ -253,7 +253,7 @@ jobs:
             script: |
               mkdir ${{ parameters.testWorkspace }}/node_modules/${{ parameters.testPackage }}/dist/test
               tar -xvf $(Pipeline.Workspace)/test-files/${{ parameters.testFileTarName }}.test-files.tar -C $(Pipeline.Workspace)/test-files
-              mv $(Pipeline.Workspace)/test-files/test/* ${{ parameters.testWorkspace }}/node_modules/${{ parameters.testPackage }}/dist/test
+              mv $(Pipeline.Workspace)/test-files/test/dist/* ${{ parameters.testWorkspace }}/node_modules/${{ parameters.testPackage }}/dist/test
```

```suggestion
              mv $(Pipeline.Workspace)/test-files/dist/test/* ${{ parameters.testWorkspace }}/node_modules/${{ parameters.testPackage }}/dist/test
```
I think they go the other way around? The older artifacts unpacked only a `test` folder (which was the contents of `dist/test`), and the new ones unpack `src` and `dist`, so what was previously found under `test/*` should now be under `dist/test/*`, right?